godtools/AppDelegate.swift

```diff
@@ -16,7 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+        MagicalRecord.setupCoreDataStack(withAutoMigratingSqliteStoreNamed: "godtools-5.sqlite")
```

Comment:
This should only happen once, right, but what do you think about moving this to a method (always on AppDelegate) and calling the method from here? (I believe we should separate the things in order to have atomic methods)